.github/workflows/scripts/register_buildpack/index.js

```diff
@@ -0,0 +1,244 @@
+const Path = require('path');
+const Schema = require('validate')
+const Toml = require('toml')
+
+const bodySchema = new Schema({
+    id: {
+        type: String,
+        required: true,
+    },
+    version: {
+        type: String,
+        match: /^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/,
+        required: true
+    },
+    addr: {
+        type: String,
+        match: /[\w][\w.-]{0,127}@sha256:[A-Fa-f0-9]{64}/,
```

Comment:
1. where does the 127 char limit come from?

Reply:
@jromero I think I also stole this from [here](https://github.com/docker/distribution/blob/master/reference/regexp.go#L37). I assume it's a limit imposed by docker?

Reply:
The addr here is pointing to `hostname/repo`.

Are we sure that the reference you pointed to isn't speaking strictly about the last component (tag) in a full registry reference. `hostname/repo:tag`. I can see that being limited but a hostname + repo path seems like it can easily go beyond 127.

Can we validate this further?

Reply:
👀

Reply:
@jromero it does appear that this regex is only for the `tag` portion.   Hmmm, need to think of a new regex.

Reply:
@elbandito here's the regex we came up with:
```
(?:[a-z0-9](?:[a-z0-9-]{0,61}[a-z0-9])?\.)+[a-z0-9][a-z0-9-]{0,61}[a-z0-9](?::[0-9]+)?\/[^:]+@sha256:[A-Fa-f0-9]{64}
```

![image](https://user-images.githubusercontent.com/475559/92162264-81e20600-edf7-11ea-8d9b-5afb69891390.png)

Reply:
Updated!